README.md

```diff
@@ -87,6 +87,24 @@ The pages below contain up-to-date build instructions:
 - [Cygwin](https://wiki.freecadweb.org/Compile_on_Cygwin)
 - [MinGW](https://wiki.freecadweb.org/Compile_on_MinGW)
 
+
+Reporting Issues
+---------
+
+To report an issue please:
+- First post to forum to verify the issue; 
+- Link forum thread to bug tracker ticket and vice-a-versa; 
+- Use the most updated stable or development versions of FreeCAD; 
+- Post version info from eg. Help>About FreeCAD>Copy to clipboard; 
```

Comment:
Please use code tags here to differentiate the text  
`Help > About FreeCAD > 'Copy to clipboard'`